test/host.test.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+
+const t = require('tap')
```

Comment:
Could you use the new `node:test` module instead?
We are migrating to it

Reply:
Sure, I'll do that